src/app/components/FinanceV2Components/utils/getAssetColor.ts

```diff
@@ -18,3 +18,21 @@ export const getAssetColor = (asset: Asset) => {
       return '#FFAC3E';
   }
 };
+export const getAssetColor2 = (asset: Asset) => {
```

Comment:
I am not sure about this naming, rename it to `getAssetSecondaryColor` or something similar.